### ISSUE TITLE
layout value not respected in directory specific data file

### ISSUE THREAD

**Issue by paulrobertlloyd** [11ty/eleventy#78]
I have a folder of posts at `./src/collections/posts/`. Inside this folder, I have the following `posts.json` file:

```
{
  "layout": "layouts/post"
}
```

However, this value (maybe others) is ignored, and so posts do not get this template applied to them. `./src/_includes/layouts/post.liquid` has the following front matter:

```
---
layout: layouts/default
permalink: '{{ page.date | date: "%Y/%m" }}/{{ title | slug }}/index.html'
---
```

I wondered if the `layout` key in this template might be causing the issue, but removing it made no difference. Bug, or user error? 😉

**Comment by chrisdmacrae** [comment]
Weighing in on this @zachleat, I'm experiencing an issue where the file isn't read at all.

E.g,

```
|- src/
|-- dirname/
|---- dirname.json
```

`dirname.json`looks like:
```
{
   "layout": "layouts/page.ejs",
   "exampleVar": "LOOK FOR ME!"
}
```

`layout` is not used by pages in this folder, nor is `exampleVar` available on any of these pages.

Using eleventy version `0.2.15`. This behaviour doesn't exist in eleventy `0.2.14`.

**Comment by zachleat** [comment]
I was able to reproduce this @paulrobertlloyd using the paths you specified. It wasn’t reading data files in the directory tree if your `--input=src` but (maybe) works if you use `--input=./src`.

Either way, this fix will roll out with 0.2.16.

**Comment by zachleat** [comment]
0.2.16 will now be 0.3.0, which shouldn’t have a functional impact to you but just fyi.
